docs/ABOUT.md

```diff
@@ -1,9 +1,19 @@
-Swift is a modern open source programming language that runs primarily on macOS and Linux. Swift was developed initially by Apple to interoperate with their key frameworks, Cocoa and CocoaTouch, both of which are mostly implemented in Objective-C.
+Swift is a modern open source programming language that runs primarily on macOS and Linux. 
```

Comment:
I am afraid this is not accurate since windows can run Swift but currently its not so easy.